DPPP/test/integration/tNDPPP.cc

```diff
@@ -21,20 +21,23 @@
 //
 // @author Ger van Diepen
 
-#include <lofar_config.h>
-#include <DPPP/DPRun.h>
 #include <casacore/tables/Tables.h>
 #include <casacore/tables/Tables/TableIter.h>
 #include <casacore/casa/Arrays/ArrayLogical.h>
 #include <casacore/casa/Arrays/ArrayMath.h>
 #include <casacore/casa/Arrays/ArrayPartMath.h>
-#include <Common/LofarLogger.h>
 #include <iostream>
 #include <stdexcept>
 
+#include <boost/test/unit_test.hpp>
+
+#include "../../DPRun.h"
+
 using namespace DP3::DPPP;
 using namespace casacore;
 
+BOOST_AUTO_TEST_SUITE(ndppp)
```

Comment:
NDPPP is the old name of the software; we should consistently use 'DPPP' now.

Reply:
I agree that we should change this then, This PR is about the unit tests and splitting the integration tests from it to fix later. I propose we leave it as is and fix this when we fix all the integration tests.

Reply:
Since this statement was added during these boost changes, I would suggest to immediately update it here. I agree we don't have to do it everywhere else in the code at this point. At many places the new name is already used, so we slowly move in that direction.